pod/src/list/list_view.rs

```diff
@@ -398,13 +398,13 @@ mod tests {
         assert_eq!(view_ro.len(), 0);
         assert_eq!(view_ro.capacity(), capacity);
         assert!(view_ro.is_empty());
-        assert_eq!(view_ro.as_slice(), &[] as &[u32]);
+        assert_eq!(&*view_ro, &[] as &[u32]);
```

Comment:
Seeing this kind of syntax gives me flashbacks to doing weird things with Arcs while learning Rust, but it's not the worst thing in the world, since someone can also call `.deref()` by hand